randovania/games/prime1/gui/dialog/prime_cosmetic_patches_dialog.py

```diff
@@ -15,18 +37,44 @@ def __init__(self, parent: QWidget, current: PrimeCosmeticPatches):
         self.setupUi(self)
         self._cosmetic_patches = current
 
+        # Build dynamically preview color squares for suits
+        suit_layouts = [ 
+            self.power_suit_color_layout, self.varia_suit_color_layout, 
+            self.gravity_suit_color_layout, self.phazon_suit_color_layout
+        ]
+        self.suit_color_preview_squares = []
+        for i, suit_colors in enumerate(SUIT_DEFAULT_COLORS):
+            color_preview_squares_for_suit = []
```

Comment:
This could be a list comprehension:
```python
self.suit_color_preview_squares.append([
    self._add_preview_color_square_to_layout(suit_layouts[i], color)
    for color in suit_colors
])
```

Reply:
Now that I look at it, the outer loop could also be but I dunno what's better to read. I leave it to your discretion

This could be a list comprehension:
```python
self.suit_color_preview_squares = [
    [
        self._add_preview_color_square_to_layout(suit_layout, color)
        for color in suit_colors
    ]
    for suit_layout, suit_colors in zip(suit_layouts, SUIT_DEFAULT_COLORS)
]
```